### PR TITLE
Add splitPills option to KvPillToggle. This style add space between t…

### PR DESCRIPTION
**Pull request by mcstover**
…he pills, allows them to wrap and ensures border radius remains consistent.

### CODE REVIEW

**Comment by ryan-ludwig** [comment]
Looks nice, at your leisure would be good to add this variant to the KvFormKitchenSink.stories.js story

**Comment by mcstover** [comment]
@ryan-ludwig Will do. Also working on a Pill style for KvRadio right now.

**Comment by mcstover** [comment]
@ryan-ludwig @mlibazisi Please have a look at the KvRadio additions.

File: src/components/Kv/KvRadio.vue [hunk 1]
```diff
@@ -109,4 +113,63 @@ export default {
...
+		flex: 1;
+		padding: rem-calc(6) rem-calc(13);
+		color: $tab-pill-color;
+		border: $tab-pill-border;
+		border-radius: $form-border-radius;
+		margin: 0;
+		font-size: 1rem;
+		text-align: center;
+		min-height: 100%;
+		margin-left: rem-calc(-1);
```
Comment (ryan-ludwig) [inline src/components/Kv/KvRadio.vue:129]: what does this negative margin achieve?

**Comment by mlibazisi** [comment]
> @ryan-ludwig @mlibazisi Please have a look at the KvRadio additions.

@mcstover will do. Thanks!

File: src/components/Kv/KvRadio.vue (hunk 1 above)
Comment (mcstover) [reply src/components/Kv/KvRadio.vue:129]: This overrides the default label margins... I can go either way with this one if we want to move that level of control out to the point of usage.

Comment (ryan-ludwig) [reply src/components/Kv/KvRadio.vue:129]: Gotcha. I think a 0 value would help things line up more consistently but don't feel strongly.

**Comment by ryan-ludwig** [comment]
> @ryan-ludwig @mlibazisi Please have a look at the KvRadio additions.

Looks good!

File: src/components/Kv/KvRadio.vue (hunk 1 above)
Comment (mcstover) [reply src/components/Kv/KvRadio.vue]: Removed negative margin

**Pull request merged**